src/App.js

```diff
@@ -1,13 +1,73 @@
 import React from 'react';
+import ToDoForm from './components/TodoComponents/TodoForm';
+import ToDoList from './components/TodoComponents/TodoList';
+import "./components/TodoComponents/Todo.css"
+
+const toDoArray = [
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      toDo: toDoArray,
+      task: "",
+      id: Date.now(),
+      completed: false
+    };
+  }
+
+  addToDo = e => {
+    e.preventDefault();
+    const newToDo = {
+      task: this.state.task,
+      id: Date.now(),
+      completed: false
+  };
+    this.setState({
+      toDo: [...this.state.toDo, newToDo],
+      task: ""
+    });
+  };
+
+  removeItem = e => {
+    e.target.classList.toggle('color')
+  }
+
+  // removeToDo = e => {
+  //   e.preventDefault();
+  //   this.setState({
+  //     toDo: [],
+  //     task: ''
+  //   })
+  // }
+
+  handleChanges = e => {
```

Comment:
```suggestion
handleChange = e => {
		const { type, name, value } = e.target;
		const val = type === 'number' ? parseFloat(value) : value;
		this.setState({ [name]: val });
	};
```

Reply:
a more verbose handleChange method that also converts any number represented as a string to an actual number.